packages/loot-core/src/server/api.ts

```
@@ -768,6 +776,166 @@
+              convertedOp = 'is';
+              break;
+            case 'isapprox':
+              convertedOp = 'isapprox';
+              break;
+            case 'isbetween':
+              convertedOp = 'isbetween';
+              break;
+            default:
+              throw APIError(`Invalid amount operator: ${value}. Expected: is, isapprox, or isbetween`);
+         `` }
+          sched._conditions[amountIndex].op = convertedOp;
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)